dev/packages/alpha/kafka/0.0.2/dataset/log/agent/stream/log.yml.hbs

```diff
@@ -0,0 +1,11 @@
+paths:
+{{#each paths as |path i|}}
+ - {{kafka_home}}{{path}}
```

Comment:
`{{kafka_home}}` here is always being replaced as an empty string, this variable is defined here: https://github.com/elastic/package-registry/pull/428/files#diff-8ddc28ec4e204b300774321f2349a083R7-R12
@mtojek @ruflin, any idea where can be the problem?

Reply:
I just played around with handlebars and to get this working it must be:
```
- {{../kafka_home}}{{path}}
```

Reply:
Interesting, thanks! I will give it a try.